### ISSUE TITLE
add ghost ping detection.

### ISSUE THREAD

**Issue by lenscas** [lenscas/thanks_bot#17]
@harshhh-dev gave me this js code that does it. Time to Rewrite it in rust
```js
const Discord = require('discord.js');
const bot = new Discord.Client();

bot.on('messageDelete', message => {
    if (!message.guild) return;
    if (message.author.bot) return;

    if (message.mentions.members.first() && message.mentions.members.first().id != message.author.id) {
        let embed = new Discord.MessageEmbed()
            .setTitle("Ghost ping detected!")
            .addField('**Author**', message.author)
            .addField("**Content**", message.content)
            .setTimestamp();
        message.channel.send(embed)
    }
})

bot.on('messageUpdate', (oldMessage, newMessage) => {
    if (!oldMessage.guild) return;
    if (message.author.bot) return;

    const oldMessageMention = oldMessage.mentions.members.first()
    const newMessageMention = newMessage.mentions.members.first()

    if (oldMessageMention && !newMessageMention && oldMessage.author.id !== oldMessageMention.id) {
        let embed = new Discord.MessageEmbed()
            .setTitle("Ghost ping detected!")
            .addField('**Author**', oldMessage.author)
            .addField("**Old Messages**", oldMessage, true)
            .addField("**New Messages**", newMessage, true)
            .setTimestamp();
        oldMessage.channel.send(embed)
    }
})
```

**Comment by lenscas** [comment]
seems like discord.js stores send messages for you, which serenity doesn't seem to do it. So... rewriting is going to take some time. #18 is a start though :)

**Comment by lenscas** [comment]
fixed by d167660cc18b3944255437561a58a43a2c71eb3e
